packages/pwa-kit-react-sdk/src/ssr/universal/contexts/index.js

```diff
@@ -61,4 +65,21 @@ CorrelationIdProvider.propTypes = {
     location: PropTypes.object
 }
 
-export {CorrelationIdContext, CorrelationIdProvider, ServerContext}
+const StorefrontPreviewProvider = ({children}) => {
+    const enabled = detectStorefrontPreview()
+    return (
+        <StorefrontPreviewContext.Provider value={{}}>
+            {/*{!isServer && enabled && (*/}
+            {/*    <Helmet>*/}
+            {/*        <script src={getClientScript()} type="text/javascript"></script>*/}
+            {/*    </Helmet>*/}
+            {/*)}*/}
+            {children}
+        </StorefrontPreviewContext.Provider>
```

Comment:
It seems this provider is unnecessary, there is no data being stored in it.
```suggestion
        <>
            {/*{!isServer && enabled && (*/}
            {/*    <Helmet>*/}
            {/*        <script src={getClientScript()} type="text/javascript"></script>*/}
            {/*    </Helmet>*/}
            {/*)}*/}
            {children}
        </>
```

Reply:
An ideal solution would have some way for users to provide `enabled` and `customizations` (what we attach to `window.STOREFRONT_PREVIEW`). Can that be done without a provider?

Reply:
we can use component props

```
const StorefrontPreview = ({enabled, customizations}) => {
  return isEnabed ? <script>...</script> : null
}
```

Reply:
Wouldn't a basic component like that require customers to make code changes to get the functionality? (i.e. wouldn't that require _everyone_ to add `<StorefrontPreview />`, not just those who want to customize?)

Is there a solution where the only setup required is installing `pwa-kit-react-sdk@latest` to have Storefront Preview work (possibly needing a file import `import '@salesforce/pwa-kit-react-sdk/storefront-preview'`), but disabling the feature can be done in _two_ lines? (e.g. import + `<StorefrontPreview enabled={false} />` or `useStorefrontPreview(false)`)